### PR TITLE
Eliminating pop-up on sign out

### PR DESCRIPTION
**Pull request by JimiC**
Added internal switch to determine the requested uri and act accordingly on the forms visibility.
Resolves issue #58.

### CODE REVIEW

**Comment by ginach** [comment]
I failed at setting up the automation to do this for me (it's fixed now but only for new PRs) but in order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's fairly quick and painless, I promise. https://cla.microsoft.com.

File: src/OneDriveSdk.WindowsForms/Web/FormsWebDialog.cs [hunk 1]
```diff
@@ -52,6 +52,15 @@ public FormsWebDialog()
...
             }
 
+            bool isSignOutRequest =
+                requestUri.AbsoluteUri.StartsWith(Constants.Authentication.MicrosoftAccountSignOutUrl,
+                    StringComparison.OrdinalIgnoreCase) ||
+                requestUri.AbsoluteUri.StartsWith(Constants.Authentication.ActiveDirectorySignOutUrl,
+                    StringComparison.OrdinalIgnoreCase);
+
+            this.ShowInTaskbar = !isSignOutRequest;
+            this.WindowState = !isSignOutRequest ? FormWindowState.Normal : FormWindowState.Minimized;
```
Comment (ginach) [inline src/OneDriveSdk.WindowsForms/Web/FormsWebDialog.cs]: Could you flip the logic here? For if/else statements it looks a bit cleaner to start with the affirmative check?

**Comment by JimiC** [comment]
Done.

**Comment by ginach** [comment]
Thanks! I realized the default branch here is wrong so I merged into dev. Closing this PR. The fix will be in the next package and I'll merge into master when we release it.

**Comment by JimiC** [comment]
Np. Didn't noticed the dev branch. Sorry for that.

**Pull request closed without merging**